data/wordlists.yml

```diff
@@ -8402,7 +8402,7 @@ Afrikaans_lower:
   categories:
   - misc
 
-[COMBO] Mail - Pass (Sql) 30858:
+Mail - Pass (Sql) 30858:
```

Comment:
Should probably include `Weakpas`. Probably not necessary to include the number of words or size of the wordlist in the wordlist name.